package.json

```diff
@@ -13,16 +13,19 @@
     "dist"
   ],
   "scripts": {
-    "build": "rm -rf dist/ && rm -rf dist.browser/ && rollup -c",
-    "dev": "rollup -c -w",
-    "build-browser": "rm -rf dist.browser/* && ./node_modules/.bin/ncc build -o dist.browser --minify",
-    "lint": "eslint \"src/**\"",
-    "lint:fix": "eslint --fix \"src/**\"",
+    "postinstall": "yarn loadConfigs",
+    "loadConfigs": "node scripts/loadConfigs.js",
```

Comment:
It would be a lot easier to understand if this script was named something like "getVersionFromPackageJSON" - the file in the scripts folder should also be named the same.

Reply:
I put this generic name because my intention is to make it easy to add other things.

(done)